src/main/java/com/perficient/movies/controllers/ActorController.java

```diff
@@ -22,23 +30,23 @@
     private ActorMapper actorMapper;
 
     @PostMapping
-    public ResponseEntity<?> createActor(@RequestBody Actor actorRequest) {
+    public ResponseEntity<?> createActor(@RequestBody ActorDto actorRequest) {
 
         actorService.createActor(actorRequest.getName(), actorRequest.getLastName(),
-                actorRequest.getBio(), actorRequest.getIsActive(), actorRequest.getBiggestRole(),
+                actorRequest.getBio(), actorRequest.isActive(), actorRequest.getBiggestRole(),
                 actorRequest.getStartYear());
 
         return new ResponseEntity<>("Actor created.", HttpStatus.CREATED);
     }
 
     @GetMapping("/{actor_id}")
     public ResponseEntity<?> findActor(@PathVariable("actor_id") Long actorId) {
-        return new ResponseEntity<>(actorMapper.toDTO((Actor) actorService.findActor(actorId)), HttpStatus.ACCEPTED);
+        return new ResponseEntity<>(actorMapper.toDto((Actor) actorService.findActor(actorId)), HttpStatus.ACCEPTED);
```

Comment:
https://stackoverflow.com/questions/55789337/best-practice-to-send-response-in-spring-boot
 you can check this practice in your API

Reply:
So, for example, in those cases I add ActorDto as the response (because of the get) and, where I only return a message I change it that for a String?